modules/core/assets/i18n/languagemenu_en.lang

```diff
@@ -0,0 +1,5 @@
+{
+  "english" : "English",
+  "german" : "German",
```

Comment:
It's fairly standard and logical to display the name of the language, in it's own language.
That way German speakers don't need to know the name of their language in English, just in German.
```suggestion
  "german" : "Deutsch",
```

Reply:
Yeah I had thought about doing that.